### PR TITLE
Updated docstrings with new draw method

### PR DESCRIPTION
**Pull request by albi3ro**
`qml.draw` has now displays slightly differently than before.  This PR updates docstrings that use `qml.draw` or `tape.draw`.

### CODE REVIEW

File: pennylane/templates/layers/random.py [hunk 1]
```diff
@@ -246,7 +237,7 @@ def compute_decomposition(
             np.random.seed(seed)
 
         shape = qml.math.shape(weights)
-        n_layers = qml.math.shape(weights)[0]
+        n_layers = shape[0]
```
Comment (albi3ro) [inline pennylane/templates/layers/random.py:240]:
```suggestion
        n_layers = qml.math.shape(weights)[0]
```

File: pennylane/templates/subroutines/interferometer.py [hunk 2]
```diff
@@ -143,14 +143,19 @@ def circuit(params):
...
 
-        The resulting circuit in this case is:
+            print(qml.draw(circuit, expansion_strategy="device")(params))
 
-        >>> print(qml.draw(circuit, expansion_strategy="device")(params))
-            0: ──R(0.713)──────────────────────────────────╭BS(0.213, 0)───R(0.681)──────────────────────────────────────────────────────────┤ ⟨I⟩
-            1: ──R(0.00912)─────────────────╭BS(0.239, 0)──╰BS(0.213, 0)───R(0.388)──────╭BS(0.622, 0)──R(0.567)─────────────────────────────┤
-            2: ──R(0.43)─────╭BS(0.534, 0)──╰BS(0.239, 0)───R(0.189)──────╭BS(0.809, 0)──╰BS(0.622, 0)──R(0.309)──╭BS(0.00845, 0)──R(0.757)──┤
-            3: ──────────────╰BS(0.534, 0)────────────────────────────────╰BS(0.809, 0)───────────────────────────╰BS(0.00845, 0)──R(0.527)──┤
+        .. code-block:: raw
```
Comment (albi3ro) [inline pennylane/templates/subroutines/interferometer.py:148]:
```suggestion
        .. code-block::
```

File: pennylane/templates/state_preparations/mottonen.py [hunk 3]
```diff
@@ -263,12 +263,16 @@ def circuit(state):
...
             state = state / np.linalg.norm(state)
 
-        The resulting circuit is:
+            print(qml.draw(circuit, expansion_strategy="device", max_length=80)(state))
 
-        >>> print(qml.draw(circuit)(state))
-            0: ──RY(2.35)──╭C─────────────╭C─────────────────╭C─────────────────────────────╭C──╭C─────────╭C──────╭C──────╭C──╭┤ State
-            1: ──RY(2.09)──╰X──RY(0.213)──╰X──╭C─────────────│───────────────╭C─────────────│───╰X─────────╰X──╭C──│───╭C──│───├┤ State
-            2: ──RY(1.88)─────────────────────╰X──RY(0.102)──╰X──RY(0.0779)──╰X──RY(0.153)──╰X───RZ(1.57)──────╰X──╰X──╰X──╰X──╰┤ State
+        .. code-block:: raw
```
Comment (albi3ro) [inline pennylane/templates/state_preparations/mottonen.py:268]:
```suggestion
        .. code-block::
```

File: pennylane/transforms/compile.py [hunk 4]
```diff
@@ -123,10 +123,17 @@ def qfunc(x, y, z):
 
         compiled_qnode = qml.QNode(compiled_qfunc, dev)
 
-    >>> print(qml.draw(compiled_qnode)(0.2, 0.3, 0.4))
-     0: ──RZ(1.57)──RX(1.57)──RZ(1.57)───RX(0.6)───────────────────────────────────────────────────────────────────────┤ ⟨Z⟩
-     1: ──RZ(1.57)──RX(1.57)──RZ(1.57)──╭X────────RZ(1.57)──────────────────────────────────────────╭C─────────────╭C──┤
-     2: ──RZ(1.57)──RX(1.57)──RZ(1.57)──╰C────────RX(0.3)───RZ(1.57)──RY(3.14)──RZ(1.57)──RY(1.57)──╰X──RY(-1.57)──╰X──┤
+        print(qml.draw(compiled_qnode)(0.2, 0.3, 0.4))
+
+    .. code-block:: raw
```
Comment (albi3ro) [inline pennylane/transforms/compile.py:128]:
```suggestion
    .. code-block::
```

File: pennylane/templates/state_preparations/mottonen.py [hunk 5]
```diff
@@ -263,12 +263,16 @@ def circuit(state):
...
             state = state / np.linalg.norm(state)
 
-        The resulting circuit is:
+            print(qml.draw(circuit, expansion_strategy="device", max_length=80)(state))
 
-        >>> print(qml.draw(circuit)(state))
-            0: ──RY(2.35)──╭C─────────────╭C─────────────────╭C─────────────────────────────╭C──╭C─────────╭C──────╭C──────╭C──╭┤ State
-            1: ──RY(2.09)──╰X──RY(0.213)──╰X──╭C─────────────│───────────────╭C─────────────│───╰X─────────╰X──╭C──│───╭C──│───├┤ State
-            2: ──RY(1.88)─────────────────────╰X──RY(0.102)──╰X──RY(0.0779)──╰X──RY(0.153)──╰X───RZ(1.57)──────╰X──╰X──╰X──╰X──╰┤ State
+        .. code-block::
```
Comment (albi3ro) [inline pennylane/templates/state_preparations/mottonen.py:268]:
```suggestion
        .. code-block::

```

**Review by Jaybsoni — APPROVED** [review]
Looks good to me! Great to see the new drawer is now fully integrated into PL.

**Pull request merged**